storeQuestionService/store-q-service.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const Question = require('./store-q-model');
+
+const app = express();
+const port = 8004;
+
+// Middleware to parse JSON in request body
+app.use(bodyParser.json());
+
```

Comment:
Ya que se está añadiendo en el fichero _package.json_ la dependencia a **Cors** se puede utilizar aquí y borrar la línea 75
```suggestion
app.use(cors());
```